debian/vzlogger.postinst

```diff
@@ -12,7 +18,7 @@ case "$1" in
 		groupmod --new-name _vzlogger vzlogger
 	fi
     if ! id _vzlogger > /dev/null 2>&1 ; then
-        adduser --quiet --system --no-create-home --home /nonexistent \
+        adduser --quiet --system "$FORCE" --no-create-home --home /nonexistent \
```

Comment:
(unlike in most other cases) you must not quote `$FORCE` here, the empty string would not be a valid parameter.

Reply:
(i prefer to use bash arrays for dynamically built commands, allowing to append argv entries conditionally, and executing via "${array[@]}", but for this simple case, unquoted expansion should do.)